docker/sqoop/scripts/rucio_contents.sh

```diff
@@ -0,0 +1,36 @@
+# Imports CMS_RUCIO_PROD.CONTENTS table for access time of datasets
+BASE_PATH=/project/awg/cms/rucio_contents/
+JDBC_URL=jdbc:oracle:thin:@cms-nrac-scan.cern.ch:10121/CMSR_CMS_NRAC.cern.ch
+if [ -f /etc/secrets/rucio ]; then
+  USERNAME=$(grep username </etc/secrets/rucio | awk '{print $2}')
+  PASSWORD=$(grep password </etc/secrets/rucio | awk '{print $2}')
+else
+  echo "Unable to read Rucio credentials"
+  exit 1
+fi
+
+# There should be always one folder
+PREVIOUS_FOLDER=$(hadoop fs -ls $BASE_PATH | awk '{ORS=""; print $8}')
+LOG_FILE=log/$(date +'%F_%H%m%S')_$(basename "$0")
+TABLE=CMS_RUCIO_PROD.CONTENTS
+TZ=UTC
+
+/usr/hdp/sqoop/bin/sqoop import \
```

Comment:
can we actually probe first if sqoop exists on a system? On vocms0181 where I do have hadoop I don't have sqoop.